### ISSUE TITLE
Version 3.4.0 breaks Chronicle-Map

### ISSUE THREAD

**Issue by andreas-hess** [OpenHFT/Chronicle-Queue#141]
Hi

I am currently investigation chronicle. Great stuff!

At the moment I have nothing but a couple of JUnit tests. After upgrading to 3.4.0 it is not possible
to create a file persisted map. Tried 2.1.0, 2.1.1 and 2.1.2.

After reverting to 3.3.6 any map version was usable again!

Andreas Heß

**Comment by BoundedBuffer** [comment]
Can you give me a bit more detail of your problem, and I can help you, between versions it best to use a unique file name. 

Rob

**Comment by andreas-hess** [comment]
Hi

Sorry for the late response; got a really bad flue.

I have tried the latest versions I've found today (queue 3.4.1 / map 2.1.4) with my test cases. Whatever has caused the issue it is gone. Looks like a supporting library incompatibility to me.

If you are interested in re-creating the issue just add queue 3.4.0 and map 2.1.2 to your class path
and try to create an instance from each. The map will not work until you revert to the queue 3.3.6
version.

Nevertheless thanks for your quick response.

Andreas
